gslib/tests/test_rsync.py

```diff
@@ -2733,7 +2733,7 @@ def test_dir_to_bucket_minus_x(self):
     @Retry(AssertionError, tries=3, timeout_secs=1)
     def _Check1():
       """Tests rsync works as expected."""
-      self.RunGsUtil(['rsync', '-d', '-x', 'obj[34]', tmpdir, suri(bucket_uri)])
+      self.RunGsUtil(['rsync', '-d', '-y', 'obj[34]', tmpdir, suri(bucket_uri)])
```

Comment:
should this be done in a loop for -x as well? Like you did it for another test down below. Ideally we would use parameterized, but I am guessing gsutil does not have that library.

Reply:
I rewrote these to be pseudo-parameterized.